api/src/pdc/providers/migration/Migrator.ts

```
@@ -228,299 +210,201 @@ export class Migrator {
+    `);
+
+    this.syncSelector(results[0]._id, territory_group.selector);
   }
 
-  async syncSelector(
-    groupId: number,
-    selector: TerritorySelectorsInterface,
-  ): Promise<void> {
+  async syncSelector(groupId: number, selector: TerritorySelectorsInterface): Promise<void> {
     const values: [number[], string[], string[]] = Object.keys(selector)
+      .filter((type) => selector && selector[type as keyof TerritorySelectorsInterface])
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)